vmm/src/cpu.rs

```diff
@@ -503,12 +503,13 @@ impl CpuManager {
         let mut vcpu_states = Vec::with_capacity(usize::from(max_vcpus));
         vcpu_states.resize_with(usize::from(max_vcpus), VcpuState::default);
 
+        let locked_device_manager = device_manager.lock().unwrap();
```

Comment:
I don't think it's really necessary to change the name. I would prefer `let device_manager = device_manager.lock().unwrap()`

Reply:
Done :)